docs/resources/convert_v1_to_v2_resource_groups.md

```diff
@@ -0,0 +1,251 @@
+
+The new Resource Groups supports an expression tree structure in which the `field` field in a filter object is defined as either `Organization ID` or `Project ID` and the `value` field contains a list of the organization or project IDs.
+
+### Example
+
+```hcl
+resource "lacework_resource_group_gcp" "example" {
+  name        = "My GCP Resource Group"
+  type        = "GCP"
+  description = "This groups a subset of Gcp Projects"
+  group {
+    operator = "AND"
+    filter {
+      filter_name = "filter1"
+      field     = "Organization ID"
+      operation = "EQUALS"
+      value     = ["MyGcpOrgID"]
+    }
+    filter {
+      filter_name = "filter2"
+      field     = "Project ID"
+      operation = "EQUALS"
+      value     = ["project-1", "project-2", "project-3"]
+    }
+  }
+}
+```
+
+# Container Resource Groups
+
+## Old Resource Groups
+
+For containers, the old Resource Groups supported the `container_tags` and `container_label` fields.
+
+### Example
+
+```hcl
+resource "lacework_resource_group_container" "example" {
+  name           = "My Container Resource Group"
+  description    = "This groups a subset of Container Tags"
+  container_tags = ["my-container"]
+  container_label {
+    key   = "name"
+    value = "my-container"
+  }
+}
+```
+
+## New Resource Groups
+
+The new Resource Groups supports an expression tree structure in which the `field` field in a filter object is defined as either `Image Tag` or `Container Label` and the `value` field contains a list of the image tags or container labels.
+
+### Example Representation
+
+```hcl
+resource "lacework_resource_group_container" "example" {
+  name        = "My Container Resource Group"
+  type        = "CONTAINER"
+  description = "This groups a subset of Container Tags"
+  group {
+    operator = "AND"
+    filter {
+      filter_name = "filter1"
+      field     = "Image Tag"
+      operation = "EQUALS"
+      value     = ["my-container"]
+   }
+   filter {
+      filter_name = "filter2"
+      field     = "Container Label"
+      operation = "EQUALS"
+      value     = ["my-container"]
+      key = “name”
+   }
+  }
+}
+```
+
+# Machine Resource Groups
+
+## Old Resource Groups
+
+For machines, the old Resource Groups supported the `machine_tags` field.
+
+### Example
+
+```hcl
+resource "lacework_resource_group_machine" "example" {
+  name        = "My Machine Resource Group"
+  description = "This groups a subset of Machine Tags"
+  machine_tags {
+    key   = "name"
+    value = "myMachine"
+  }
+}
+```
+
+## New Resource Groups
+ new Resource Groups supports an expression tree structure in which the `field` field in a filter object is defined as either `Image Tag` or `Container Label` and the `value` field contains a list of the image tags or container labels.
+ 
+The new Resource Groups supports an expression tree structure in which the `field` field in a filter object is defined as `Machine Tag` and the `value` field contains a list of the machine tags.
```

Comment:
You have container related info here duplicated. I think you copy pasted and forgot to delete.